.ci/docker/centos-rocm/Dockerfile

```diff
@@ -100,6 +100,18 @@ COPY ./common/install_ninja.sh install_ninja.sh
 RUN if [ -n "${NINJA_VERSION}" ]; then bash ./install_ninja.sh; fi
 RUN rm install_ninja.sh
 
+ARG TRITON
+# Install triton, this needs to be done before sccache because the latter will
+# try to reach out to S3, which docker build runners don't have access
+ENV CMAKE_C_COMPILER cc
```

Comment:
Not sure why this is required and since it is set in the ENV it will effect other component builds.
Not sure if this will cause any side effects.
However ok to proceed with current changes. LGTM.